js.js

```
@@ -0,0 +1,52 @@
+        if (i * j === 5000) {
+            console.log("Found it!");
+        }
+    }
+}
+
+// Insecure input handling - possible XSS vulnerability
+document.getElementById("output").innerHTML = "User: " + window.location.search;
+
+// Inefficient array handling
+var arr = [1, 2, 3, 4, 5];
+for (var i = 0; i < arr.length; i++) {
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)